apps/web/components/landing/AssetsSection.tsx

```
@@ -11,35 +11,15 @@ import {
-    symbol: 'USDC',
-    name: 'USD Coin',
-    region: 'Global',
-    paymentMethods: 'Varies',
-    color: 'bg-emerald-600',
-  },
-];
-
 export function AssetsSection() {
   const reducedMotion = useReducedMotion();
+  const supportedAssets = getSupportedAssets();
+  const regionCount = new Set(supportedAssets.map((a) => a.region)).size;
```

Biome (2.3.13)

[error] This variable regionCount is unused.

Unused variables are often the result of typos, incomplete refactors, or other sources of bugs.
Unsafe fix: If this is intentional, prepend regionCount with an underscore.


(lint/correctness/noUnusedVariables)